algosdk/atomic_transaction_composer.py

```diff
@@ -698,8 +704,52 @@ def __init__(
 
 class AtomicTransactionResponse:
     def __init__(
-        self, confirmed_round: int, tx_ids: List[str], results: ABIResult
+        self,
+        confirmed_round: int,
+        tx_ids: List[str],
+        results: ABIResult,
+        tx_infos: Dict[str, dict] = None,
+        methods: Dict[int, abi.Method] = None,
     ) -> None:
         self.confirmed_round = confirmed_round
         self.tx_ids = tx_ids
         self.abi_results = results
+        self.tx_infos = tx_infos
+        self.methods = methods
+
+    def transactions_trace(self, quote='"') -> str:
```

Comment:
Proposing the ability to print out an execution trace of an atomic transaction with inner apps. However, to attain parity across all our SDK's for this feature may be too much of a pain. I'll revert if the consensus is that I should.